internal/controlplane/server_test.go

```
@@ -151,6 +151,66 @@ func TestServerDeletesOnlyTerminalJobsWithSubmissionAuthorization(t *testing.T)
 	}
 }
 
+func TestServerDeletesOnlyDisconnectedIdleWorkers(t *testing.T) {
+	server, webServer := newTestHTTPServer(t)
+	defer webServer.Close()
+	if response := postJSON(t, webServer.URL+"/api/v1/workers/poll", pollRequest("worker-old", []string{"test"}, []string{"machinist"}), map[string]string{"Authorization": "Bearer secret"}); response.StatusCode != http.StatusOK {
+		response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)